src/main/java/com/minecolonies/coremod/network/messages/GuardMobAttackListMessage.java

```diff
@@ -0,0 +1,92 @@
+package com.minecolonies.coremod.network.messages;
+
+import com.minecolonies.api.util.BlockPosUtil;
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.colony.ColonyView;
+import com.minecolonies.coremod.colony.buildings.AbstractBuildingGuardsNew;
+import com.minecolonies.coremod.colony.buildings.views.MobEntryView;
+import io.netty.buffer.ByteBuf;
+import jline.internal.Nullable;
+import net.minecraft.util.math.BlockPos;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessage;
+import net.minecraftforge.fml.common.network.simpleimpl.MessageContext;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GuardMobAttackListMessage extends AbstractMessage<GuardMobAttackListMessage, IMessage>
+{
+    /**
+     * The position of the building.
+     */
+    private BlockPos buildingId;
+
+    /**
+     * The colony the building is within.
+     */
+    private int colonyId;
+
+    /**
+     * List of mobsToAttack.
+     */
+    private List<MobEntryView> mobsToAttack = new ArrayList<>();
+
+    /**
+     * Empty standard constructor.
+     */
+    public GuardMobAttackListMessage()
+    {
+        super();
+    }
+
+    public GuardMobAttackListMessage(final int colonyId, final BlockPos buildingId, final List<MobEntryView> mobsToAttack)
+    {
+        super();
+        this.colonyId = colonyId;
+        this.buildingId = buildingId;
+        this.mobsToAttack = mobsToAttack;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Store a copy of "mobsToAttack". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2384)